portal/component-tests/contract/components/contract-overview-table.component-test.js

```diff
@@ -78,11 +81,7 @@ describe(component, () => {
       wrapper
         .expectText('[data-cy="submissionDate"]')
         .toRead(
-          formatInTimeZone(
-            new Date(dealWithManualInclusionApplicationSubmissionDate.details.manualInclusionApplicationSubmissionDate),
-            LONDON_TIMEZONE,
-            FULL_DATE,
-          ),
+          formatInTimeZone(new Date(dealWithManualInclusionApplicationSubmissionDate.details.manualInclusionApplicationSubmissionDate), LONDON, FULL_DATE),
```

Comment:
Whilst we're in here, any chance we can destructure the date so this reads nicer? 

```js
formatInTimeZone(new Date(manualInclusionApplicationSubmissionDate), LONDON, FULL_DATE),
```

We could also rename `dealWithManualInclusionApplicationSubmissionDate` to `dealWithMiaSubmissionDate`

Appreciate this isn't the goal of the PR but whilst we're here, would be great